lib/rspec/sql.rb

```diff
@@ -57,6 +57,15 @@ def matcher
       @matcher
     end
 
+    def diff(actual, expected)
+      if expected.is_a?(Numeric)
+        change = actual - expected
+        format("%+d", change)
+      else
+        Expectations.differ.diff_as_object(actual, expected)
+      end
+    end
```

Comment:
I guess `differ` doesn't provide a method for integers, that's sad 😞

Reply:
Yes, I searched through a lot of RSpec code and couldn't find any differ doing this.